chapters/ch07.md

```diff
@@ -1,27 +1,27 @@
 # How large do the dev/test sets need to be?
 
 ->
-# Tập phát triển / đánh giá cần lớn đến mức nào?
+# Tập phát triển/kiểm tra cần lớn đến mức nào?
 
 The dev set should be large enough to detect differences between algorithms that you are trying out. For example, if classifier A has an accuracy of 90.0% and classifier B has an accuracy of 90.1%, then a dev set of 100 examples would not be able to detect this 0.1% difference. Compared to other machine learning problems I’ve seen, a 100 example dev set is small. Dev sets with sizes from 1,000 to 10,000 examples are common. With 10,000 examples, you will have a good chance of detecting an improvement of 0.1%. [2]
 
 ->
-Tập phát triển phải đủ lớn để có thể thấy được sự khác nhau giữa các thuật toán mà bạn đang thử. Ví dụ: nếu thuật toán phân loại A có độ chính xác 90,0% và thuật toán phân loại B có độ chính xác 90,1%, thì một tập có 100 ví dụ sẽ không thể phát hiện sự khác biệt 0,1% này. So với các vấn đề khác trong machine learning mà tôi đã thấy, một tập 100 ví dụ là nhỏ. Các tập phát triển với kích thước từ 1.000 đến 10.000 ví dụ khá phổ biến. Với 10.000 ví dụ, bạn sẽ có khả năng cao để nhìn thấy mức cải thiện 0,1%. [2]
+Tập phát triển phải đủ lớn để có thể thấy được sự khác nhau giữa các thuật toán mà bạn đang thử. Ví dụ: nếu thuật toán phân loại A có độ chính xác 90,0% và thuật toán phân loại B có độ chính xác 90,1%, thì một tập có 100 mẫu sẽ không thể phát hiện sự khác biệt 0,1% này. So với các vấn đề khác trong machine learning mà tôi đã thấy, một tập phát triển với 100 mẫu là nhỏ. Các tập phát triển với kích thước từ 1.000 đến 10.000 mẫu khá phổ biến. Với 10.000 mẫu, bạn sẽ có khả năng cao để nhìn thấy mức cải thiện 0,1%. [2]
 
 For mature and important applications—for example, advertising, web search, and product recommendations—I have also seen teams that are highly motivated to eke out even a 0.01% improvement, since it has a direct impact on the company’s profits. In this case, the dev set could be much larger than 10,000, in order to detect even smaller improvements.
 
 ->
-Trong các ứng dụng quan trọng và đã khá phát triển - lấy ví dụ như quảng cáo, tìm kiếm trên web và đề xuất sản phẩm - tôi đã thấy nhiều nhóm có động lực cao để cải thiện thậm chí chỉ là 0,01%, vì nó có ảnh hưởng trực tiếp đến lợi nhuận của công ty. Trong trường hợp này, tập phát triển có thể lớn hơn 10.000, để phát hiện được những cải thiện còn nhỏ hơn nữa.
+Trong các ứng dụng quan trọng và đã khá phát triển - lấy ví dụ như quảng cáo, tìm kiếm trên web và gợi ý sản phẩm - tôi đã thấy nhiều nhóm rất muốn cải thiện thậm chí chỉ là 0,01%, vì nó có ảnh hưởng trực tiếp đến lợi nhuận của công ty. Trong trường hợp này, tập phát triển có thể lớn hơn 10.000 để phát hiện được những cải thiện còn nhỏ hơn nữa.
```

Comment:
Mình đề xuất sửa như sau 
- Có thể thêm từ "(chất lượng) thuật toán" để rõ nghĩa hơn. Vì cải thiện có thể là thời gian chạy, năng lực tính toán. 
>  tôi đã thấy nhiều nhóm rất muốn cải thiện chất lượng thuật toán 
- thêm từ "nhiều" với nghĩa nhấn mạnh của "much larger"
- Gợi ý sửa cho mượt hơn 

> Trong trường hợp này, tập phát triển có thể lớn hơn 10.000 mẫu rất nhiều chỉ để đạt mức cải tiến còn nhỏ hơn nữa.

Reply:
* "mature app" -> có thể là "ứng dụng đã đưa vào khai thác" vì nó "trưởng thành" rồi có thể sử dụng được rồi.

> Với những ứng dụng quan trọng và đã đưa vào khai thác như tìm kiếm web,...

* "profits" có thể là "lợi ích" của công ty không? Vì nó có nghĩa đạt được nhiều thứ hơn là lợi nhuận

Reply:
Thanks all.